docs/README.md

```
@@ -4,13 +4,21 @@ Welcome to the Containerization Assistant MCP Server documentation.
 
 ## Quick Navigation
 
+### User Guides
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)